docusaurus/docs/advanced-configuration.md

```diff
@@ -3,21 +3,17 @@ id: advanced-configuration
 title: Advanced Configuration
 ---
 
-The `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a plugin.
+The plugin `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a Grafana plugin. Although you can make changes, we recommend against doing so. Instead, follow the guidance in this topic to customize your tooling configs.
```

Comment:
I understand the change of voice but IMO this is not warning enough that they must not edit the `.config` folder. The original language clearly stayed "do not edit". the "at your own risk" sounds more like a legal term and us devs usually work better with explicit text.

Reply:
Fixed.